prover/cmd/prover/cmd/setup.go

```diff
@@ -32,7 +33,8 @@ import (
 type SetupArgs struct {
 	Force      bool
 	Circuits   string
-	DictPath   string
+	DictPath   string // to be deprecated; only used for compiling v0 blob decompression circuit
```

Comment:
call it `DictPathV0Deprecated` ?

Reply:
The thing is, it is technically not deprecated since it still has a legitimate use-case that the new alternative doesn't cover (v0 circuits).